backend/models/slo-definition.js

```
@@ -0,0 +1,169 @@
+            alertThreshold: 0.25,
+        },
+        {
+            serviceId: 'notification-service',
+            serviceName: 'Notification Service',
+            targetAvailability: 99.5,
+            trackingWindow: '7days',
+            alertThreshold: 0.25,
+        },
+    ];
+
+    demoSLOs.forEach(slo => create(slo));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)